smarts/sstudio/genhistories.py

```diff
@@ -178,10 +179,10 @@ def create_output(self, time_precision=3):
                     float(self.column_val_in_row(row, "sim_time")) / 1000,
                     time_precision,
                 ),
-                float(self.column_val_in_row(row, "position_x") + x_offset)
-                * self.scale,
-                float(self.column_val_in_row(row, "position_y") + y_offset)
-                * self.scale,
+                float(self.column_val_in_row(row, "position_x"))
+                + x_offset * self.scale,
+                float(self.column_val_in_row(row, "position_y"))
+                + y_offset * self.scale,
```

Comment:
Why was there a need to do this?

Reply:
Good catch!!
A change was required there for when `self.column_val_in_row()` returns a string (which happens for Interaction datasets).  This can't be "added" to `x_offset` or `y_offset` until it is cast to a float first.  However, I screwed up and didn't keep the outer parentheses by accident.

I'm glad you caught this!!